rustfmt.toml

```diff
@@ -14,7 +14,7 @@ ignore = [
     # to be fixed shortly
     "src/libcore/iter/adapters/mod.rs",
     "src/libcore/iter/traits/iterator.rs",
-    "src/librustc/hir/lowering.rs",
+    "src/librustc_lowering/lib.rs",
```

Comment:
This got pretty frustrating not being able to do `^C + S` -- would be good to nix the tidy line length check soon so we can fix this.